opm/simulators/wells/GasLiftStage2.cpp

```diff
@@ -832,7 +832,7 @@ computeDelta(const std::string& well_name, bool add)
         if (this->well_state_.wellIsOwned(well.indexOfWell(), well_name)) {
             const auto& well_ecl = well.wellEcl();
             Scalar factor = well_ecl.getEfficiencyFactor() *
-                            this->well_state_[well_name].efficiency_scaling_factor;
+                            this->well_state_.getGlobalEfficiencyScalingFactor(well_name);
```

Comment:
Are you sure there was a problem here? 
Line 832 should prevent processing for wells that are not owned by us via `if (this->well_state_.wellIsOwned(well.indexOfWell(), well_name))`

Maybe we could use this at the other places, too?

Reply:
You're right - got a bit too eager here.. will revert.